src/styles/_orderModal.scss

```diff
@@ -0,0 +1,25 @@
+.order-modal {
```

Comment:
🟡 For performance in Next.js projects, leaning more heavily on Tailwind CSS is often beneficial. The styling in _orderModal.scss can be achieved using Tailwind's utility classes, it would likely be a lighter option.